### ISSUE TITLE
Unity 5.3.0, Editor crash when object has TouchScript Component and Camera is on scene

### ISSUE THREAD

**Issue by FUBAR2077** [TouchScript/TouchScript#197]
I have a reproductable problem - if you setup a scene with TouchScript, EventSystem, Camera, and GameObject with any TouchScript gesture component, check the object with component so it will be wisible in inspector, and play and unplay scene, the Unity Editor crashes. Any ideas what causes the problem and how to deal with it?

![yavnuk](https://cloud.githubusercontent.com/assets/4373019/11960125/53bf0108-a8d1-11e5-9adf-d7e380eae35f.jpg)

**Comment by valyard** [comment]
Indeed. I'll submit a bug report.

**Comment by FUBAR2077** [comment]
any progress on topic?

**Comment by martinmaurer** [comment]
I was not able to pinpoint the specific cause of the crashes,. The problem was somehow connected to the Touchscript TUIO input module (which is receiving over UDP Port 3333). 

Meanwhile touchscript 7.0 was released on github (it doesn’t seem to be updated on the Appstore yet). It has a minimum requirement of unity 5.1 so I figured that maybe some compatibility issues with current versions have been fixed, (while the TUIO module doesn’t seem to have been touched at all)
Well, in the end it runs stable now. Thank God for the open Source community, this project is due in five days :-)

Best,
-martin
